pkg/terminal/command.go

```diff
@@ -2676,7 +2677,15 @@ func printfile(t *Term, filename string, line int, showArrow bool) error {
 		arrowLine = line
 	}
 
-	file, err := os.Open(t.substitutePath(filename))
+	var file *os.File
+	path := t.substitutePath(filename)
+	if _, err := os.Stat(path); errors.Is(err, os.ErrNotExist) {
```

Comment:
There's people around using old versions of go, let's just use `os.IsNotExist` here.

Reply:
go.mod explicitly says `go 1.16` so people on older versions can't use this regardless. I think it should be fine?

https://github.com/go-delve/delve/blob/master/go.mod#L3

Reply:
The go version in go.mod is not the minimum supported version of Go, delve does build on versions older than 1.16. It's true that we haven't bothered to keep track of how low we can get but errors.Is vs os.IsNotExist is too small of a difference to compromise.

Reply:
Ack, I'll change it :)